shared/settings/notifications/index.js.flow

```diff
@@ -25,6 +25,8 @@ export type Props = {
   onToggleUnsubscribeAll: (group: string) => void,
   mobileHasPermissions: boolean,
   waitingForResponse: boolean,
+  sound?: boolean,
+  onToggleSound?: (sound: boolean) => void,
```

Comment:
It looks like these can be `sound: boolean` and `onToggleSound: (sound: boolean) => void`. They are never `undefined` right?

Reply:
They can be undefined on mobile. Flow complained about this otherwise.

Reply:
Ah I see it's under the `!isMobile` gate. LGTM then!